blog/2022-11-09-nginx-monitoring.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: nginx-monitoring
+title: "How to monitor NGINX web servers?"
+description: "How to monitor NGINX web servers?"
+image: https://user-images.githubusercontent.com/96257330/200813970-c1cb5be7-21ec-4365-8cfd-bef9545197f1.png
+tags: [how-to,infrastructure-monitoring,nginx,web-servers]
+keywords: [how-to,infrastructure-monitoring,nginx,web-servers]
+authors: satya
+---
+Web servers are among the most important components in modern IT infrastructures. They host the websites, web services, and web applications that we use on a daily basis. Social networking, media streaming, software as a service (SaaS), and other activities wouldn’t be possible without the use of web servers. And with the advent of cloud computing and the movement of more services online, web servers and their monitoring are only becoming more important. Given the extensive usage of Web servers, Sysadmins and SREs should monitor web servers as a key aspect for performance. 
+
+<!--truncate-->
+
+## What is NGINX?
+
+NGINX (pronounced “engine X”) is a popular HTTP server and reverse proxy server. As an HTTP server, NGINX serves static content very efficiently and reliably, using relatively little memory. As a reverse proxy, it can be used as a single, controlled point of access for multiple back-end servers or for additional applications such as caching and load balancing. NGINX is available as a free, open source product or in a more full-featured, commercially distributed version called NGINX Plus.
```

Comment:
```suggestion
[NGINX](https://www.nginx.com/) (pronounced “engine X”) is a popular [HTTP server](https://en.wikipedia.org/wiki/Web_server) and [reverse proxy](https://en.wikipedia.org/wiki/Reverse_proxy) server. As an HTTP server, NGINX serves static content very efficiently and reliably, using relatively little memory. As a reverse proxy, it can be used as a single, controlled point of access for multiple back-end servers or for additional applications such as [caching](https://en.wikipedia.org/wiki/Web_cache) and [load balancing](https://en.wikipedia.org/wiki/Load_balancing_(computing)). NGINX is available as a free, open source product or in a more full-featured, commercially distributed version called NGINX Plus.
```